### PR TITLE
feature(superinput): replace rawExpressionInput's input with superinput

### PR DESCRIPTION
**Pull request by Raid55**
This PR adds superinput to the studio, starting with the rawExpression input on the `ConditionalModalForm`. This is a draft PR until I can get studio working on next to check this PR is 💯

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "author": "Botpress, Inc.",
   "license": "AGPL-3.0",
   "scripts": {
-    "build": "yarn workspaces foreach --exclude @botpress/superinput -ptiv run build",
+    "build": "yarn workspaces foreach -ptiv run build",
     "watch": "yarn workspaces foreach --exclude @botpress/superinput -piv run watch",
     "clean": "yarn workspaces foreach --exclude @botpress/superinput -pi run clean",
```
Comment (laurentlp) [inline package.json:11]: Can you also remove the `--exclude @botpress/superinput` for those two scripts too

File: packages/studio-ui/package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "@analytics/segment": "^1.1.3",
     "@blueprintjs/core": "^3.23.1",
     "@blueprintjs/select": "^3.12.0",
+    "@botpress/superinput": "workspace:^",
```
Comment (laurentlp) [inline packages/studio-ui/package.json:26]: Why the caret?

**Review by laurentlp — COMMENTED** [review]
Small comments otherwise LGTM

File: packages/studio-ui/package.json (hunk 2 above)
Comment (Raid55) [reply packages/studio-ui/package.json:26]: I just ran 'yarn workspace @botpress/studio-ui add @botpress/superinput' I'm just it's just to keep installing the latest build or something

**Review by laurentlp — APPROVED** [review]

**Pull request merged**